docs/3.0/deploy/infrastructure-concepts/workers.mdx

```diff
@@ -102,6 +102,34 @@ Workers have two statuses: `ONLINE` and `OFFLINE`. A worker is online if it send
 If a worker misses three heartbeats, it is considered offline. By default, a worker is considered offline a maximum of 90 seconds 
 after it stopped sending heartbeats, but you can configure the threshold with the `PREFECT_WORKER_HEARTBEAT_SECONDS` setting.
 
+### Worker logs
+<Warning>This feature is experimental as of Prefect version 3.X.X</Warning>
+
+<span class="badge cloud"></span>Workers can now send logs directly to Prefect Cloud. To enable this feature:
+
+1. Set the environment variable:
+
+```bash
+PREFECT_EXPERIMENTS_WORKER_LOGGING_TO_API_ENABLED=True
```

Comment:
should we prefix this with `export` or `prefect config set`?

Reply:
What's your advice?

Reply:
That is, between exporting an env var and setting a config. If a worker is running in infrastructure, I think they'll need to set the env var, but if it's a process worker running locally then they could set it via `config`?

Reply:
I took a shot!